components/script/dom/webidls/StyleSheet.webidl

```diff
@@ -7,8 +7,8 @@
 interface StyleSheet {
   readonly attribute DOMString type_;
   readonly attribute DOMString? href;
+  readonly attribute Element? ownerNode;
```

Comment:
As we can see from the [spec](https://drafts.csswg.org/cssom/#stylesheet), the type of `ownerNode` is still `Element or ProcessingInstruction`. I think it's still good to keep the type here.

```
  readonly attribute (Element or ProcessingInstruction)? ownerNode;
```

@emilio how do you think?

Reply:
I think given Servo doesn't support XML processing instructions, the difference is meaningless. All browsers use `Node?` in the IDL as you noted on IRC... So all of them are fine really.

Reply:
Yes, after checking [Chrome's implementation](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/css/style_sheet.h?q=ownerNode&sq=package:chromium&g=0&l=46) and [Firefox's implementation](https://searchfox.org/mozilla-central/source/layout/style/StyleSheet.h#257), both of them use `Node?` type for `ownerNode`. So, let's keep `Element?` here because Servo doesn't support XML processing instructions 🙇